### ISSUE TITLE
SymbolSource.org NuGet target fails on .symbol push - considering removing .symbol packages

### ISSUE THREAD

**Issue by Aaronontheweb** [akkadotnet/akka.net#839]
This is the issue that tripped up #838 - SymbolSource.org has barfed up the following error during the past two releases:

[22:17:25][Step 1/1] Your package was pushed.
[22:17:25][Step 1/1] Pushing Akka 1.0.0 to the symbol server (http://nuget.gw.symbolsource.org/Public/NuGet)...
[22:17:28][Step 1/1] Failed to process request. 'Package submission failed: The length of the string value exceeds the length configured in the mapping/parameter. Exception: System.Exception: Error while processing project: 'Akka'. ---> System.Exception: Error while processing version: '1.0.0'. ---> System.Exception: Error while processing compilation: 'Release' '.NETFramework,Version=v (...). See http://www.symbolsource.org/Public/Home/Help for possible reasons. Fiddler may help diagnosing this error if your client discards attached detailed information.'. 
[22:17:28][Step 1/1] The remote server returned an error: (506) Package submission failed: The length of the string value exceeds the length configured in the mapping/parameter. Exception: System.Exception: Error while processing project: 'Akka'. ---> System.Exception: Error while processing version: '1.0.0'. ---> System.Exception: Error while processing compilation: 'Release' '.NETFramework,Version=v (...). See http://www.symbolsource.org/Public/Home/Help for possible reasons. Fiddler may help diagnosing this error if your client discards attached detailed information...

TL;DR; - there's something messed up on their end and I don't know what it is. The issue appears to be intermittent.

This error is stopping us from being able to upload symbol packages period. Should we do away with them altogether? Contact them (I've tried) and figure out what the cause of the error is and if we can work around it? Use an alternative?

**Comment by Silv3rcircl3** [comment]
Is this still an issue? 
Seems to be working for the nightly builds except two times.

**Comment by Aaronontheweb** [comment]
Still comes up every now and then - had this issue when pushing a DotNetty release a couple of weeks ago. So it's not specific to Akka.NET, but we should try to work around it with some more granular catch...retry logic inside our FAKE script.

**Comment by Aaronontheweb** [comment]
Closed via #1492
